README.md

```diff
@@ -136,6 +136,7 @@ The primary focus of this list is to provide alternatives that prioritize privac
 	- [Droid-ify](https://github.com/Iamlooker/Droid-ify) - Lightweight F-Droid client with Material UI.
```

Comment:
Droid-ify already listed

Reply:
Oh yes, I don't know how I didn't see it, my bad!